analyzer/src/main/kotlin/managers/utils/GoSupport.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2022 EPAM Systems, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.analyzer.managers.utils
+
+object GoSupport {
```

Comment:
We don't wrap functions that could be top-level in an `object` in the other support files, so I wouldn't do so here either. The "namespace" for top-level functions simply is the file name itself.

Reply:
But like I wrote in the other comment, if this is the only function to introduce this file for, I'd be fine with keeping it at `GoMod`.

Reply:
> We don't wrap functions that could be top-level in an `object` in the other support files, so I wouldn't do so here either. 

Hm, just because we pollute the namespace elsewhere we have to follow that approach.
What advantage do you see in putting these into an object? ..and why do you think the advantage is worth less than (global) consistency in this case?

>The "namespace" for top-level functions simply is the file name itself.

Isn't the namespace for non-private top level functions the Java / Kotlin package?

Reply:
Done anyway.